docs/user/Dependencies.md

```diff
@@ -4,55 +4,53 @@ If your environment requires validated support for a specific version or range o
 
 The following list details our dependencies and versions (~ indicates multiple possible sources):
 
-SOURCE | NAME | VERSION
---- | --- | ---
-~ | python | 3.x
-~ | pip | 3.x
-~ | unzip | any
-~ | tar | any
-~ | docker | any
-~ | git | any
-~ | kustomize | 3.7.0
-~ | kubectl | 1.21 - 1.23
-~ | AWS IAM Authenticator | 1.18.9/2020-11-02
-~ | Helm | 3
-pip3 | ansible | 2.10.7
-pip3 | openshift | 0.12.0
-pip3 | kubernetes | 12.0.1
-pip3 | dnspython | 2.1.0
-ansible-galaxy | community.kubernetes | 1.2.1
+| SOURCE | NAME | VERSION |
+| --- | --- | --- |
+| ~ | python | 3.x |
+| ~ | pip | 3.x |
+| ~ | unzip | any |
+| ~ | tar | any |
+| ~ | docker | any |
+| ~ | git | any |
+| ~ | kustomize | 3.7.0 |
+| ~ | kubectl | 1.21 - 1.23 |
+| ~ | Helm | 3 |
+| pip3 | ansible | 2.10.7 |
+| pip3 | openshift | 0.12.0 |
+| pip3 | kubernetes | 12.0.1 |
+| pip3 | dnspython | 2.1.0 |
+| pip3 | docker | 5.0.3 |
+| ansible-galaxy | community.kubernetes | 1.2.1 |
+| ansible-galaxy | community.docker | 2.7.0 |
+| ansible-galaxy | ansible.utils | 2.3.0 |
 
 Required project dependencies are generally pinned to known working or stable versions to ensure users have a smooth initial experience. In some cases it may be required to change the default version of a dependency. In such cases users are welcome to experiment with alternate versions, however compatibility may not be guaranteed.
 
 # Docker
 
 If deploying via the [Dockerfile](../../Dockerfile) overriding a dependency version can be accomplished by supplying one or more docker build arguments:
 
-ARG | NOTE
---- | ---
-kustomize_version | the version of kustomize to use
-kubectl_version | the version of kubectl to use
-aws_iam_authenticator_version | the version of aws iam authenticator to use
-pip_ansible_version | the version of ansible to use from pip
-pip_openshift_version | the version of openshift to install from pip
-pip_kubernetes_version | the version of kubernetes to install from pip
-pip_dnspython | the version of dnspython to install from pip
-ansible_galaxy_community_kubernetes_version | the version of community.kubernetes to install from ansible-galaxy
-ansible_galaxy_ansible_posix_version | the version of ansible.posix to install from ansible-galaxy
+| ARG | NOTE |
+| --- | --- |
+| kustomize_version | the version of kustomize to use |
+| kubectl_version | the version of kubectl to use |
+| aws_cli_version | the version of AWS CLI to use |
+| gcp_cli_version | the version of Google cloud SDK to use |
+| HELM_VERSION | the version helm to use |
 
 As described in the [Docker Installation](./DockerUsage.md) section add additional build arguments to your docker build command:
 
 ```bash
 docker build \
-	--build-arg pip_openshift_version=0.12.0 \
+	--build-arg kustomize_version=3.7.0 \
```

Comment:
What was the reason in changing the example here? The kustomize_version should already be 3.7.0 and this seems to lend itself to the fact that default is not sufficient.

_Originally posted by @thpang in https://github.com/sassoftware/viya4-deployment/pull/224#discussion_r939029463_

Reply:
@thpang This line here looks like it was added to demonstrate that the user can set build args to override versions when building the docker image. I changed it from `pip_openshift_version` to `kustomize_version` because `pip_openshift_version` is no longer ARG in the dockerfile.
It was removed by Norm on Apr 1, 2021, this doc is just outdated.
See https://github.com/sassoftware/viya4-deployment/commit/50590215c369b63b08f36f87f5d43c6011a699b2

I can replace the example with `kubectl_version` instead since that is variable the user is more likely to replace.

Reply:
Changed this to kubectl_version instead.